spec/controllers/assignment_invitations_controller_spec.rb

```diff
@@ -67,6 +67,45 @@
       end
     end
 
+    context 'github repository already exists' do
+      let(:assignment_repo) { AssignmentRepo.new(assignment: assignment, user: user) }
+
+      before do
+        assignment_repo.create_github_repository
+      end
+
+      it 'creates a new assignment repo with a name suffix' do
```

Comment:
Split this into two `it` blocks with one `expect` each.

1. 'creates a new assignment repo'
2. 'new repository name has expected suffix'